gcp-py-cloudrun-cloudsql/__main__.py

```diff
@@ -19,16 +19,16 @@
     "database", instance=cloud_sql_instance.name, name=config.require("db-name")
 )
 
-users = pulumi_gcp.sql.User(
-    "users",
+user = pulumi_gcp.sql.User(
+    "user",
     name=config.require("db-name"),
     instance=cloud_sql_instance.name,
     password=config.require_secret("db-password"),
 )
 
 sql_instance_url = Output.concat(
     "postgres://",
-    config.require("db-name"),
+    user.name,
```

Comment:
Even though `config.require("db-name") === user.name`, it'd be best to reference `user.name` instead so the chain of dependencies is clear to both the reader and to the Pulumi engine.